src/Routing/ValidatesRequests.php

```diff
@@ -20,7 +20,7 @@
     public function validate(Request $request, array $rules, array $messages = [], array $customAttributes = [])
     {
         $validator = $this->getValidationFactory()->make($request->all(), $rules, $messages, $customAttributes);
-
+         $validator->setAttributeNames($customAttributes);
```

Comment:
Because when you use ValidatesRequests trait in the controller,if you want change the value of attributes,like this:
```php
$attributes = [
            'name' => '姓名',
            'mobile' => '手机号码',
            'qq' => 'QQ号码',
            'company_name' => '公司名称',
            'company_tel' => '公司电话'
        ];
$this->validate($this->request, $rules, [], $attributes);
```
it's not working